infrastructure/main.tf

```diff
@@ -110,3 +110,56 @@ resource "azurerm_key_vault_secret" "draft-store-key" {
   value        = random_string.draft_encryption_key.result
   key_vault_id = data.azurerm_key_vault.ccd_shared_key_vault.id
 }
+
+////////////////////////////////
+// DB version 11             //
+////////////////////////////////
+
+module "data-store-db-v11" {
+  source          = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product         = "${var.component}-db-v11"
```

Comment:
can't we just use `var.product`?

Reply:
This is going to be used for the DB name.  we would like to keep it with different values that the original DB definition module